Formula/maturin.rb

```diff
@@ -0,0 +1,21 @@
+class Maturin < Formula
+  desc "Build and publish Rust crates as Python packages"
+  homepage "https://github.com/PyO3/maturin"
+  url "https://github.com/PyO3/maturin/archive/refs/tags/v0.10.6.tar.gz"
+  sha256 "e6a9a67cc62ffe248654e60e7ec211bf23319c4c936ad87022f7a1fd0997430d"
+  license "MIT"
+  head "https://github.com/PyO3/maturin.git"
+
+  depends_on "rust" => :build
```

Comment:
```suggestion
  depends_on "rust"
```
Based on the new test, I assume `maturin` requires Rust at runtime.